stl/inc/compare

```diff
@@ -363,8 +363,9 @@ namespace _Strong_order {
     void strong_order(); // Block unqualified name lookup; see GH-1374.

```

Comment:
Now that the Standard mandates this, these comments (here and below) can be simplified to `// Block unqualified name lookup`. I've updated GH-1374 to note this ADL overhaul (although it still tracks an issue regarding value categories).